openapi/workday/client.bal

```diff
@@ -0,0 +1,195 @@
+    string httpVersion = "1.1";
+    # Configurations related to HTTP/1.x protocol
+    http:ClientHttp1Settings http1Settings = {};
+    # Configurations related to HTTP/2 protocol
+    http:ClientHttp2Settings http2Settings = {};
+    # The maximum time to wait (in seconds) for a response before closing the connection
+    decimal timeout = 60;
+    # The choice of setting `forwarded`/`x-forwarded` header
+    string forwarded = "disable";
+    # Configurations associated with Redirection
+    http:FollowRedirects? followRedirects = ();
+    # Configurations associated with request pooling
+    http:PoolConfiguration? poolConfig = ();
+    # HTTP caching related configurations
+    http:CacheConfig cache = {};
+    # Specifies the way of handling compression (`accept-encoding`) header
+    http:Compression compression = http:COMPRESSION_AUTO;
+    # Configurations associated with the behaviour of the Circuit Breaker
+    http:CircuitBreakerConfig? circuitBreaker = ();
+    # Configurations associated with retrying
+    http:RetryConfig? retryConfig = ();
+    # Configurations associated with cookies
+    http:CookieConfig? cookieConfig = ();
+    # Configurations associated with inbound response size limits
+    http:ResponseLimitConfigs responseLimits = {};
+    # SSL/TLS-related options
+    http:ClientSecureSocket? secureSocket = ();
+|};
+
+# This is a generated connector for [WorkDay Customer Accounts REST API v1](https://community.workday.com/sites/default/files/file-hosting/restapi/index.html) OpenAPI specification.
+# The customer accounts service enables applications to access  customer accounts information including customers, invoices, adjustments, and payments.
+@display {label: "Workday Customer Accounts", iconPath: "“resources/workday.customeraccounts.svg”"}
+public isolated client class Client {
+    final http:Client clientEp;
+    # Gets invoked to initialize the `connector`.
+    # The connector initialization requires setting the API credentials.  Follow [this guide] https://credentials.workday.com/docs/authentication/ and obtain tokens.
+    #
+    # + clientConfig - The configurations to be used when initializing the `connector` 
+    # + serviceUrl - URL of the target service 
+    # + return - An error if connector initialization failed 
+    public isolated function init(ClientConfig clientConfig, string serviceUrl) returns error? {
+        http:Client httpEp = check new (serviceUrl, clientConfig);
+        self.clientEp = httpEp;
+    }
+    # Retrieves a single customer invoice payment remittance line instance.
+    #
+    # + iD - The Workday ID of the resource. 
+    # + subresourceID - The Workday ID of the subresource. 
+    # + return - Successful response. A successful response can return no matched data. 
+    remote isolated function getInvoicePaymentRemittanceLine(string iD, string subresourceID) returns Viewremittancedetails17b89f3dcfab1000056c7a877cbe00f1|error {
```

Comment:
`string iD` seems odd to use as a parameter name. Shall we rename the parameter name to `id` as below in OpenAPI file ? Please look in other remote functions too. 
```
      - name: id
        in: path
        description: The Workday ID of the resource.
        required: true
        schema:
          type: string
```